PR for json-schema-comments (#3338)
PrairieLearn/PrairieLearn: docs/course.md

    "uuid": "cef0cbf3-6458-4f13-a418-ee4d7e7505dd",
    "name": "TAM 212",
    "title": "Introductory Dynamics",
    "comment": "The assessment set order used here will be the one shown within PrairieLearn",
    "assessmentSets": [
        {"abbreviation": "HW", "name": "Homework", "heading": "Homeworks", "color": "green1"},
        {"abbreviation": "E", "name": "Exam", "heading": "Exams", "color": "red1"}
    ],

>     "comment": "Topics should be thought of as chapters or sections in a textbook",

Reviewer: While I like these examples, I don't think we can support multiple comments within the same object.

Reviewer 2: resolved in 502ad59f3

Property | Description
--- | ---

> `comment` | The comments for each property.

Reviewer 3: Instead, do the following:

- move the section `##Timezone` before `## Colors`
- between `Timezone` and `Colors`, add a section named `## Comment`

## Comment

You can add comments to `json` files using the property `comment`. You can use this property more than once in the same file. For example:

```json
{
    "comment": "select the timezone for the course",
    "timezone": "America/New_York"
}
```

> `comment` | The comments for each property.

Reviewer 3: I fixed the comment above

```

Allowable timezones are those in the TZ column in the [list of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), which is a display version of the [IANA Time Zone Database](https://www.iana.org/time-zones).

> ## Comment

Reviewer: ```suggestion
## Comments in JSON files
```

Reviewer 2: included in 502ad59f3

> You can add comments to json files using the property `comment`. You can use this property more than once in the same file. For example:

Reviewer: ```suggestion
You can add comments to JSON files using the property `comment` on any object. You can only use this property once for each object. For example:
```

```json
{
    "comment": "assessments that are shared among a group of students",
    "assessmentSets": [
      {"abbreviation": "GA", "name": "Group Activity", "heading": "Group Activities", "color": "green1"}
    ],

>     "comment": "timezone for the course",

Reviewer: We should delete this the second comment here. Although PL currently allows more than one comment per object, it actually silently drops all the comments except the last one if the file is automatically edited.

I don't currently have a good way to catch JSON files with repeated keys (like more than one comment in the same object). `JSON.parse()` silently drops all earlier copies of a key. As does `jju` that we use.

Reviewer 2: Applied.

    "timezone": "America/New_York"
}
```

> ## Colors

Reviewer: Why was this shifted?
